dashboard/app/models/plc/written_enrollment_task_assignment.rb

```diff
@@ -1,44 +0,0 @@
-# == Schema Information
```

Comment:
My understanding is that the review status will be tracked by the underlying level, but I can add a note in Trello to track.